docs/content/preview/explore/_index.md

```diff
@@ -226,6 +226,53 @@ ycqlsh>
   </div>
 </div>
 
+## Set up YB Simulation Base Demo
+
+YB Simulation Base Demo is a Java application that simulates workloads against YugabyteDB. Some Explore topics use the application to demonstrate features of YugabyteDB.
+
+For more information about the workload application, refer to [YB Simulation Base Demo](https://github.com/yugabyte/yb-simulation-base-demo-app/).
+
+### Install
+
+YB Simulation Base Demo requires Java 11 or later installed on your computer.
+
+Download the Simulation Base Demo JAR file (`yb-simu-base-app.jar`) using the following command:
+
+```sh
+wget https://github.com/yugabyte/yb-simulation-base-demo-app/releases/download/1.0/yb-simu-base-app.jar
+```
+
+### Run
+
+To start the application against a running local cluster, use the following command:
+
+```sh
+java -jar ./yb-simu-base-app.jar
+```
+
+By default, the application connects to the local cluster at 127.0.0.1.
+
+To connect to a different address or node, use the `-Dnode` flag to specify an IP address. For example:
+
+```sh
+java -jar ./yb-simu-base-app.jar -Dnode=127.0.0.2
+```
+
+To view the application UI, navigate to <http://localhost:8080>.
+
+### Start a workload
+
+To start a workload, do the following:
+
+1. In the application UI, Click the hamburger icon beside Active Workloads for Generic.
+1. Select the **Usable Operations** tab.
+1. Under **Create Tables**, click **Run Create Tables Workload** to add tables to the database.
+1. Under **Seed Data**, click **Run Seed Data Workload** to add data to the tables.
+1. Under **Simulation**, click **Run Simulation Workload**.
```

Comment:
We need to enable **Include new Inserts** and then **Run Simulation Workloads**. By default the app performs only Reads.

Reply:
Under **Simulation**, enable **Include new Inserts** and select **Run Simulation Workload** to start a simulation that performs read and write operations across all the nodes of your cluster.